appinventor/blocklyeditor/src/blocks/components.js

```diff
@@ -170,6 +170,13 @@ Blockly.Blocks.component_event = {
     if (this.instanceName == oldname) {
       this.instanceName = newname;
       this.componentDropDown.setValue(this.instanceName);
+      if (this.isCollapsed())
+      {
+        var COLLAPSED_INPUT_NAME = '_TEMP_COLLAPSED_INPUT';     
```

Comment:
Thanks, I'll change that.

I used the same name as in line 1597 of ```/appinventor/lib/blockly/src/core/block.js``` - I thought it might be suitable as it's referring to the same string.